.github/workflows/pypi-release.yml

```diff
@@ -5,9 +5,9 @@ jobs:
   pypi_release:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v4
     - name: Set up Python 3.x
-      uses: actions/setup-python@v2
+      uses: actions/setup-python@v3
```

Comment:
Any reason to not use `actions/setup-python@v4` ?

Reply:
My best guess: When I looked up the latest version number, and noted it down wrong, I wasn't wearing my glasses or the sun was in my eyes or I was drunk. :-( Sorry.